src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+const Book = () => (
```

Comment:
- [ ] Please, make sure this "Book" component takes the book data as a _prop_ and renders the title and the author. 

## Explanation and hints:

### Example of how to create the new `Book` component that takes the book data as props:

```JAVASCRIPT
const Book = (props) => {
  const {
    id, genre, title, author, currentChapter,
  } = props;
```

That way, your components will communicate with each other, ready for the data, and for the next steps! 💪 

I hope this is helpful.

_These are just examples of one way to do it based on my own experience and effort. Included them here just to help and give you an idea, and should not be taken as an "official" way to do it. For this milestone, you only need the title and the author._

Reply:
Thank you so much for your guidance, this cleared a lot of doubts.